src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -208,13 +208,11 @@ private void ReplicateToDestination()
                         {
                             while (true)
                             {
-#if DEBUG
```

Comment:
@ppekrol Are we running all of our tests in Debug mode, right?
As we have lots of Assert.Debug which should be tested.